src/liboslexec/llvm_ops.cpp

```diff
@@ -642,14 +642,14 @@ OSL_SHADEOP void osl_transformn_vmv(void *result, void* M_, void* v_)
 {
    const Vec3 &v = VEC(v_);
    const Matrix44 &M = MAT(M_);
-   M.inverse().transpose().multDirMatrix (v, VEC(result));
+   M.inverse().transposed().multDirMatrix (v, VEC(result));
```

Comment:
This is Imath::M44f, it's a method. So I can't just add multDirMatrixT without also breaking symmetry in a weird way.

If only Bjarne had won the member-vs-function battle!